### ISSUE TITLE
Use safer eval

### ISSUE THREAD

**Issue by williamrs-openai** [oughtinc/ice#39]
I would suggest using a safer version of eval by default in the [Interpreters](https://primer.ought.org/chapters/tool-use/interpreters) page (e.g. worry about generated code making web requests).

[asteval](https://newville.github.io/asteval/) looks simple to use for mathematical expressions, which is all the tutorial requres.

**Comment by stuhlmueller** [comment]
I agree! I think in practice this doesn't matter right now but good to do.
